DuckDuckGo/SettingsViewModel.swift

```diff
@@ -461,12 +457,12 @@ extension SettingsViewModel {
         if AppDependencyProvider.shared.vpnFeatureVisibility.isPrivacyProLaunched() {
             switch connectionStatus {
             case .connected:
-                self.state.networkProtection.status = UserText.netPCellConnected
+                self.state.networkProtectionConnected = true
             default:
-                self.state.networkProtection.status = UserText.netPCellDisconnected
```

Comment:
Both `UserText.netPCellConnected` and `UserText.netPCellDisconnected` are not used anywhere else and can be removed from `UserText`.

Reply:
removed